robottelo/utils/ohsnap.py

```diff
@@ -2,13 +2,21 @@
 import requests
 from box import Box
 from packaging.version import Version
+from wait_for import wait_for
 
 from robottelo import constants
 from robottelo.exceptions import InvalidArgumentError
 from robottelo.exceptions import RepositoryDataNotFound
 from robottelo.logging import logger
 
 
+def ohsnap_response_hook(r, *args, **kwargs):
+    """Requests response hook callback function that processes the response"""
+    if not r.ok:
+        logger.warn(f'Request to: {r.request.url} got response: {r.status_code}')
```

Comment:
warn is a deprecated method, can we use warning?
```suggestion
        logger.warning(f'Request to: {r.request.url} got response: {r.status_code}')
```

Reply:
ah darn, thought it was the other way around :D

Reply:
done